### DISCUSSION TITLE
Make fzf to keep the history of only accepted/valid commads

### DISCUSSION THREAD

**Discussion by TayyabAsghar** [junegunn/fzf#2464] [Ideas]
We can find all the commands on fzf. But I think it would be better if we make it to accept the valid commands only. Like I use PowerLevel10k so it would show me cross if I enter the wrong command. This could be because of misspelling. fzf stores that command, but it should not store it.

**Comment by junegunn** [comment]
> fzf stores that command

It is not stored by fzf. fzf simply filters the command history your shell maintains.

```sh
fc -lr 1 | fzf
```

You need to find a way to modify how your shell manages the history.

**Reply by TayyabAsghar** [reply]
Got it.
